data/com.vixalien.decibels.metainfo.xml.in.in

```diff
@@ -30,7 +30,7 @@
   <content_rating type="oars-1.1" />
   <releases>
     <release version="0.1.3" date="2023-09-24">
-      <description>
+      <description translatable="no">
```

Comment:
Is there any reason the release notes shouldn't be translatable?

Reply:
As I said on the commit: "GNOME automatically excludes release descriptions on Damned Lies (GNOME Translation Platform). It's a good practice to follow the GNOME way.

This can streamline the translation process, allowing translators to focus their efforts on more critical and user-facing aspects of the application."

Also, release notes tend to be much more than UI strings.

Reply:
okay